### DISCUSSION TITLE
Fonts

### DISCUSSION THREAD

**Discussion by Kayto** [Kayto/Armachat-circuitpython#8] [General]
I have not tried this but some notes from Bobricius.......you can try in armachat code to enable better fonts...

font
font_file = "fonts/neep-iso8859-1-12x24.bdf"
font_file = "fonts/gohufont-14.bdf"
font_file = "fonts/Gomme10x20n.bdf"
font = bitmap_font.load_font(font_file)
font = terminalio.FONT
 
you get extra line, and more clean text, I am not using this becauseit extent startup loading time then in screen function modify  text_scale=1

**Comment by Kayto** [comment]
The latest code on this fork has been updated with some font changes by Bobricius.
This enables display of line 8, by changing font and scale

1. The message screen text size may be too small in which case change text scale back to 2 in line 665 of code.py in this fork.

**Reply by Kayto** [reply]
above issue fixed.
The "wip_code.py" provides improved fonts and better readability.

**Reply by richteel** [reply]
I may have overwritten some changes you had in code.py regarding fonts but I did attempt to keep most of it. I was having the text only being displayed on the top half of the screen so I changed some things. You may want to go back and take a look if you attempted to make any of these changes in code.py.

**Reply by Kayto** [reply]
Thanks. I think I have this covered there was a mising reference to the fonts in the editor part of the code which caused the very small half screen problem!
